smcontroller/smcontroller.go

```diff
@@ -103,6 +105,12 @@ func New(
 		nodes:                     make(map[string]*smHandler),
 	}
 
+	controller.logHandler = map[string]func(cloudprotocol.RequestLog) error{
+		cloudprotocol.SystemLog:  controller.getSystemLog,
+		cloudprotocol.ServiceLog: controller.getServiceLog,
```

Comment:
I would rename `ServiceLog` to `InstanceLog` it would be more generic. Need to discuss with Cloud team.